os/arch/arm/src/amebasmart/amebasmart_i2s.h

```diff
@@ -52,7 +52,9 @@ extern "C" {
 */
 typedef enum {
 	I2S_NUM_0 = 0x0,			/*!< I2S 0 */
-	//I2S_NUM_1 = 0x1,			/*!< I2S 1 */
+	I2S_NUM_1 = 0x1,			/*!< I2S 1 */
```

Comment:
You removed I2S0 and I2S1 from the Kconfig. Is this necessary to leave them?

Reply:
The current adaptor index is aligned with the MAX adaptors, this is done to prevent confusion in the future. Would this be fine with you?

Reply:
Do you mean that I2S0 and I2S1 will be added later?

Reply:
We have 4 adaptors, I2S0 and I2S1 are internal I2S without GPIO pins which currently will not used based on the current requirement. If there is a need, then we will add.  Do you prefer to remove instead?

Reply:
if there are 4 i2s ports, it should be defined. By KConfig or something, some of those should be selected.

Reply:
There are reasons for this, the internal I2S will not be able to use TX/RX as I2S0 & I2S1 externally as there are no external pins and used internally for our audio Sport. Therefore I2S0 & I2S1 cannot be used, this is a limitation. We were hoping to directly link the porting layer of (i2s2) to our driver (i2s2). 
If this is confusing, we can change to porting layer (i2s0) (i2s1) to link our driver layer (i2s2) (i2s3) respectively.

Reply:
Based on discussion, we will remove I2S_NUM_0 & I2S_NUM_1, since it does not impact the code flow in anyway.